cli-feed-v4.json

```diff
@@ -627,6 +709,123 @@
           "default": "true"
         }
       ]
+    },
+    "3.22.2": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "netcore3": {
+            "displayInfo": {
+              "displayName": ".NET Core 3",
+              "hidden": false,
+              "displayVersion": "v3",
+              "targetFramework": ".NET Core",
+              "description": "LTS"
+            },
+            "capabilities": "",
+            "sdk": {
+              "name": "Microsoft.NET.Sdk.Functions",
+              "version": "3.0.11"
+            },
+            "default": true,
+            "localEntryPoint": "func.exe",
+            "targetFramework": "netcoreapp3.1",
+            "toolingSuffix": "",
```

Comment:
empty for .NET Core 3 inproc

Reply:
We should opt for not having the property instead of empty string.

Reply:
I think @phil-allen-msft suggested to define empty. Phil, do you prefer if we don't have the property at all?

Reply:
Thank you! I will remove this property from .NET Core 3 entries then.